stdlib/asm/math/ec_ext5.masm

```diff
@@ -714,3 +714,511 @@ export.mul.10
     push.0.0.0.0
     loc_loadw.5
 end
+
+#! Given a 319 -bit scalar ( say e ) on stack, this routine computes elliptic curve point 
+#! b s.t. b =  e * G, using double-and-add technique | G = conventional group generator point.
+#!
+#! Group generator point https://github.com/pornin/ecgfp5/blob/ce059c6/rust/src/curve.rs#L67-L83
+#!
+#! Scalar e should be lesser than N ( = 1067993516717146951041484916571792702745057740581727230159139685185762082554198619328292418486241 ).
+#! Note, scalar e should be provided as 10 limbs on stack, each of 32 -bit, representing it in radix-2^32 form.
+#! 
+#! Given a 319 -bit scalar e, following python code snippet should convert it to 
+#! desired input form i.e. radix-2^32 representation, having ten 32 -bit limbs
+#!
+#! [(e >> (32*i)) & 0xffff_ffff for i in range(10)]
+#!
+#! Expected stack state
+#!
+#! [e0, e1, e2, e3, e4, e5, e6, e7, e8, e9, ...]
+#!
+#! Scalar e = (e0, e1, e2, e3, e4, e5, e6, e7, e8, e9)
+#!
+#! Final stack state
+#!
+#! [x0, x1, x2, x3, x4, y0, y1, y2, y3, y4, inf, ...]
+#!
+#! Point b = (x, y, inf) | b = e * G
+#!
+#! See https://github.com/itzmeanjan/secp256k1/blob/cbbe199/point.py#L174-L186 for source of inpiration.
+export.gen_mul.8
```

Comment:
To be consistent with the `secp256k1` module, should we have the other procedure to be `point_mul` (rather than just `mul`)?

Reply:
Well actually in latest state of secp256k1 curve arithmetic implementation, we've

`mul` : For scalar multiplication of arbitrary group element ( See [here](https://github.com/itzmeanjan/miden/blob/0c3f8a84593eb21aa03ba1ab532d3772966a1bf3/stdlib/asm/math/secp256k1_ec.masm#L942-L980) )
`gen_mul` : For scalar multiplication of group generator element ( See [here](https://github.com/itzmeanjan/miden/blob/0c3f8a84593eb21aa03ba1ab532d3772966a1bf3/stdlib/asm/math/secp256k1_ec.masm#L1174-L1208) )

So we're consistent.